composer.json

```diff
@@ -168,6 +168,7 @@
         "drupal/cache_metrics": "^2",
         "drupal/ckeditor_liststyle": "^1.4",
         "drupal/coffee": "^1",
+        "drupal/color": "^1.0",
```

Comment:
Is this deliberate? Its disabled in this PR in core.extensions.yml.

Reply:
Yes, I've updated environment indicator module, and it removed/disabled color module. So i guess we don't need it. Also it is deprecated and will be removed in Drupal 10, so I think this is a good step forward to the upgrade

Reply:
This PR is *adding* color module to composer.json

Reply:
Updating the env_indicator removes the module, I've re-added to avoid issues. We need to initally disable it before removing. That's why I've added it back to be able to safely disable the color module from the configs.

Reply:
Makes sense. Feel free to resolve conflict and merge.